client/components/button/style.scss

```diff
@@ -159,6 +159,7 @@ button {
 
 .button.is-borderless {
 	border: none;
+	background: none;
```

Comment:
>Could this have nay side effects?

Doubtful, but maybe. Would mean we're relying on borderless buttons having a white background, which I don't think we should be, and seems an oversight of not stripping some of the default button styles (not caught because in most cases it's on a white background anyways).